contracts/scripts/whitelist.ts

```diff
@@ -1,23 +1,43 @@
 // Import ethers from Hardhat package
 const { ethers } = require("hardhat");
 
-async function main() {
+const oracleContractAddress = "0x9615049fbd3d5Ae3efB83501a251aB661c165505"
```

Comment:
Well, did not do a intelligent CLI argument insertion, will edit these every time.
Also tested if the full attestation fits nicely to a tx - did not face any issues.